### PR TITLE
Update RSVP and ticket block initial start date to be published date

### PR DESCRIPTION
**Pull request by nealfennimore**
https://central.tri.be/issues/117844
https://central.tri.be/issues/117845

### CODE REVIEW

File: src/modules/data/blocks/rsvp/sagas.js [hunk 1]
```diff
@@ -66,16 +68,16 @@ export function* setRSVPTempDetails( action ) {
 }
 
 export function* initializeRSVP() {
-	const start = yield select( blocks.datetime.selectors.getStart );
-	const end = yield select( blocks.datetime.selectors.getEnd );
+	const publishDate = wpSelect( 'core/editor' ).getEditedPostAttribute( 'date' );
+	const eventEnd = yield select( blocks.datetime.selectors.getEnd );
```
Comment (paulmskim) [inline src/modules/data/blocks/rsvp/sagas.js]: shouldn't this be `eventStart` and `selectors.getStart`?

File: src/modules/data/blocks/ticket/sagas.js [hunk 2]
```diff
@@ -251,14 +251,16 @@ export function* setTicketInitialState( action ) {
 		dateIsPristine: get( 'dateIsPristine', DEFAULT_TICKET_STATE.dateIsPristine ),
 	};
 
-	const start = yield select( blocks.datetime.selectors.getStart );
-	const end = yield select( blocks.datetime.selectors.getEnd );
+	const publishDate = wpSelect( 'core/editor' ).getEditedPostAttribute( 'date' );
+	const eventEnd = yield select( blocks.datetime.selectors.getEnd );
```
Comment (paulmskim) [inline src/modules/data/blocks/ticket/sagas.js]: same question as above

File: src/modules/data/blocks/rsvp/sagas.js (hunk 1 above)
Comment (nealfennimore) [reply src/modules/data/blocks/rsvp/sagas.js]: That's what I originally thought too, but according to this ticket it's supposed to be the publish date of the event initially

Comment (nealfennimore) [reply src/modules/data/blocks/rsvp/sagas.js]: https://central.tri.be/issues/117844

Comment (paulmskim) [reply src/modules/data/blocks/rsvp/sagas.js]: had a conversation with gustavo earlier, he confirmed it's the end date.

Comment (nealfennimore) [reply src/modules/data/blocks/rsvp/sagas.js]: Ah shoot, I misread. Fixing

Comment (nealfennimore) [reply src/modules/data/blocks/rsvp/sagas.js]: Fixed here: https://github.com/moderntribe/event-tickets/pull/912

**Pull request merged**